omod/src/main/resources/apps/overallActions_extension.json

```diff
@@ -29,5 +29,15 @@
         "icon": "icon-link",
         "order": 13,
         "requiredPrivilege": "Task: coreapps.mergeVisits"
+    },
+    {
+        "id": "${project.parent.groupId}.${project.parent.artifactId}.deletePatient",
+        "extensionPointId": "patientDashboard.overallActions",
+        "type": "script",
+        "label": "coreapps.task.deletePatient.label",
+        "script": "delPatient.showDeletePatientCreationDialog({{patient.uuid}})",
```

Comment:
Check the browser console for javascript errors. If there are none, debug REST WS module to see what's doing (maybe there's a stack trace in the log).